net/mdns-repeater/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       mdns-repeater
+PLUGIN_VERSION=    1.0
+PLUGIN_COMMENT=    Start, stop, configure mdns-repater
```

Comment:
this could explain better what it does, like "Allow multicast DNS between networks" maybe?

Reply:
good idea